nixpkgs_review/cli/merge.py

```diff
@@ -6,4 +6,4 @@
 
 def merge_command(args: argparse.Namespace) -> None:
     github_client = GithubClient(ensure_github_token(args.token))
-    github_client.merge_pr(get_current_pr())
+    github_client.merge_pr(get_current_pr(), dict(method=args.method))
```

Comment:
Have you tested if that works? It's called `merge_method` here: https://docs.github.com/en/rest/reference/pulls#merge-a-pull-request

Reply:
```
curl \
  -X PUT \
  -H "Accept: application/vnd.github.v3+json" \
  https://api.github.com/repos/octocat/hello-world/pulls/42/merge \
  -d '{"commit_title":"commit_title"}'
```